default.nix

```diff
@@ -107,7 +107,7 @@ let
   create.partition = q: x: let
     env = helper.device-id q.device;
   in ''
-    parted -s "''${${env}}" mkpart ${x.part-type} ${x.fs-type or ""} ${x.start} ${x.end}
+    parted -s "''${${env}}" mkpart ${x.part-type or ""} ${x.fs-type or ""} ${x.start} ${x.end}
```

Comment:
```suggestion
    parted -s "''${${env}}" mkpart ${x.part-type or ""} ${x.fs-type or "primary"} ${x.start} ${x.end}
```

Reply:
From the arch wiki (https://wiki.archlinux.org/title/Parted), part-type is only needed when using MBR.

Reply:
@Mic92 I believe the man page you have linked is outdated. This is the current parted man page which specifies the optionality of part-type (https://www.gnu.org/software/parted/manual/parted.html#mkpart)